docs/actions-swift.md

```diff
@@ -187,164 +187,7 @@ wsk action update helloSwiftly action-bin.zip --kind swift:4.2
 wsk action invoke helloSwiftly -r
 ```
 
-## Compiling Swift 4.1 packaged actions
```

Comment:
should this be removed or does it make sense to update this for 4.2? or maybe replace with https://github.com/apache/incubator-openwhisk-runtime-swift#packaging-an-action-as-a-swift-executable-using-swift-4x ?

Reply:
The 3.1 steps should be removed. We can keep 4.x

Reply:
actually there's already a section: `Compiling a single source file for Swift 4.2` and `Compiling dependencies and multi-file projects for Swift 4.2` so what you've removed is fine.